packages/eds-core-react/stories/playground/DataTable.tsx

```diff
@@ -112,7 +113,48 @@ const toCellValues = (data: Data[], columns: Column[]) =>
 
 const MenuButton = ({ row }: { row: string[] }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement>()
+  const [focus, setFocus] = useState<MenuProps['focus']>(null)
   const isOpen = Boolean(anchorEl)
+
+  const openMenu = (
+    focus: MenuProps['focus'],
+    e: React.KeyboardEvent<HTMLButtonElement>,
+  ) => {
+    setFocus(focus)
+    setAnchorEl(e.currentTarget)
+  }
+  const closeMenu = () => {
+    setFocus(null)
+    setAnchorEl(null)
+  }
+
+  const onKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+    const { key } = e
+    if (key === 'Tab') {
+      return
+    }
+    e.preventDefault()
+    e.stopPropagation()
+
+    switch (key) {
+      case 'Enter':
+        isOpen ? closeMenu() : openMenu('first', e)
+        break
+      case 'ArrowDown':
+        if (isOpen) {
```

Comment:
The arrowDown and ArrowUp cases do nothing and can either be removed, or fixed so they open the menu on first or last item